core/blockchain/impl/block_tree_error.cpp

```diff
@@ -0,0 +1,29 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include "blockchain/block_tree_error.hpp"
+
+OUTCOME_CPP_DEFINE_CATEGORY(kagome::blockchain, BlockTreeError, e) {
+  using E = kagome::blockchain::BlockTreeError;
+  switch (e) {
+    case E::INVALID_DB:
+      return "genesis block is not provided, and the database is either empty "
+             "or does not contain valid block tree";
+    case E::NO_PARENT:
+      return "block, which was tried to be added, has no known parent";
```

Comment:
I believe it would be more correct to write something like
```suggestion
      return "block, which should have been added, has no known parent";
```